locale/zh_CN.po

```diff
@@ -7849,7 +7851,7 @@ msgstr "勾选使用从右到左文字布局。"
 
 #: ../src/richtext/richtextfontpage.cpp:320
 msgid "Suppress hyphe&nation"
-msgstr "抑制断字 (&n)"
+msgstr "抑制断字"
```

Comment:
```suggestion
msgstr "抑制断字(&N)"
```

Reply:
fixed in .po

Reply:
![Screen Shot 2022-07-11 at 11 00 58](https://user-images.githubusercontent.com/82932766/178180798-e5a8cec2-55db-4128-90a2-b335b4c4bbf3.png)

Reply:
comment revised.
![Screen Shot 2022-07-11 at 11 21 44](https://user-images.githubusercontent.com/82932766/178182695-a818e767-33a7-4aaa-a273-bcf36e053571.png)